src/main/java/com/solvd/carina/demo/mobile/saucedemomobile/android/ProductsScreen.java

```diff
@@ -0,0 +1,60 @@
+package com.solvd.carina.demo.mobile.saucedemomobile.android;
+
+import com.solvd.carina.demo.mobile.saucedemomobile.common.CartScreenBase;
+import com.solvd.carina.demo.mobile.saucedemomobile.common.LeftSideMenuScreenBase;
+import com.solvd.carina.demo.mobile.saucedemomobile.common.ProductDetailsScreenBase;
+import com.solvd.carina.demo.mobile.saucedemomobile.common.ProductsScreenBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = ProductsScreenBase.class)
+public class ProductsScreen extends ProductsScreenBase {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @FindBy(xpath = "//android.view.ViewGroup[@content-desc=\"test-Cart drop zone\"]//android.widget.TextView[@text='PRODUCTS']")
+    private ExtendedWebElement screenTitle;
+
+    @FindBy(css = "//android.view.ViewGroup[@content-desc='test-Cart']/android.view.ViewGroup/android.widget.ImageView")
```

Comment:
```suggestion
    @FindBy(css = "//android.view.ViewGroup[@content-desc='test-Cart']")
```
please check if this locator works